core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXUpgradeV1Controller.java

```diff
@@ -80,6 +81,12 @@ public DistroXUpgradeV1Response upgradeClusterByCrnInternal(@ResourceCrn String
                 dataHubOsUpgradeEntitled));
     }
 
+    @Override
+    @InternalOnly
+    public DistroXCcmUpgradeV1Response upgradeCcmByCrnInternal(String crn) {
```

Comment:
missing annotation from parameter (https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#how-to-make-an-api-internally-callable)